frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -95,8 +134,18 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
     return true;
   };
 
+  const namespaceStoreNextConditions = () => {
+    if (state.namespacePolicyType === NamespacePolicyType.SINGLE) {
+      return state.readNamespaceStore.length === 1 && state.writeNamespaceStore.length === 1;
+    }
+    return false;
+  };
+
   const creationConditionsSatisfied = () => {
-    if (!backingStoreNextConditions()) return false;
+    if (state.bucketClassType === BucketClassType.STANDARD) {
+      if (!backingStoreNextConditions()) return false;
```

Comment:
if i add `!backingStoreNextConditions` to this condition and it fails. It will jump to next else if and fail always. This may block the whole creation process. 

i'll keep this the same.